app/presenters/sub_section_json_presenter.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+class SubSectionJsonPresenter
+  HEADER_PATTERN = PatternMaker.call(
+    "starts_with hashes then perhaps_spaces then capture(title) and nothing_else",
+    hashes: "#+",
+    title: '\w.+',
+  )
+
+  LINK_PATTERN = PatternMaker.call(
+    "starts_with perhaps_spaces within(brackets,capture(label)) then perhaps_spaces and within(sq_brackets,capture(url))",
+    label: '\s*\w.+',
+    url: '\s*(\b(https?)://)?[-A-Za-z0-9+&@#/%?=~_|!:,.;]+[-A-Za-z0-9+&@#/%=~_|]\s*',
+  )
+  # Url pattern from https://stackoverflow.com/a/163684/1014251
+
+  attr_reader :sub_section
+
+  def initialize(sub_section)
+    @sub_section = sub_section
+  end
+
+  delegate :title, to: :sub_section
+
+  def output
+    {
+      details: {
+        sections: [
+          {
+            title: title,
+            sub_sections: sub_sections,
+          },
+        ],
+      },
+    }
+  end
+
+  def errors
+    @errors ||= []
```

Comment:
So we basically swallow the error? So if a user enters bad markdown, there's no validation running on that, so it will bomb out when we try to parse it. The result will be that the draft content item will be missing a subsection. That seems ok I think.

Reply:
Should we send the errors somewhere? eg Sentry?

Reply:
Yes - that's how it behaves. I think we can add the handling of errors later (on publish perhaps)